test/index.test.js

```diff
@@ -64,12 +66,6 @@ describe('Outbound', function() {
     });
   });
 
-  describe('loading', function() {
-    it('should load', function(done) {
-      analytics.load(outbound, done);
-    });
-  });
```

Comment:
is this a required method for integrations to include when integrating with segment?